lib/foreman_tasks/dynflow/daemon.rb

```diff
@@ -41,20 +42,22 @@ def run_background(command = "start", options = {})
 
       STDERR.puts("Dynflow Executor: #{command} in progress")
 
-      Daemons.run_proc(options[:process_name],
+      options[:executors_count].times do
+        Daemons.run_proc(options[:process_name],
```

Comment:
Each daemon needs to have unique process name. I'm getting the following error
```
bundle exec ../foreman-tasks/bin/dynflow-executor -c 2 start
Dynflow Executor: start in progress
dynflow_executor: process with pid 348 started.
ERROR: there is already one or more instance(s) of the program running
```

Editing the code to add the current iteration to the process name I got it to spawn several executors. The downside is the user has to remember how many executor he has if he wants to stop them. I'd expect stop would stop all executors
```
# bundle exec ../foreman-tasks/bin/dynflow-executor -c 2 start
Dynflow Executor: start in progress
dynflow_executor-0: process with pid 7266 started.
dynflow_executor-1: process with pid 7278 started.

# ps -e -o pid,cmd | grep executor
 7264 dynflow_executor-0_monitor
 7266 dynflow_executor-0
 7278 dynflow_executor-1
 7282 dynflow_executor-1_monitor

# bundle exec ../foreman-tasks/bin/dynflow-executor stop                                                                                                                                   
Dynflow Executor: stop in progress
dynflow_executor-0: trying to stop process with pid 7266...
dynflow_executor-0: process with pid 7266 successfully stopped.

# ps -e -o pid,cmd | grep executor                      
 7278 dynflow_executor-1
 7282 dynflow_executor-1_monitor
```